common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -65,10 +68,74 @@ innovation_small_soldiers = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
 #culture_group_regional
 ####################
 ##Tribal##
 ####################
+innovation_waterspeaking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @waterspeaking
+	potential = {
+		##Jinyu
+		this.culture_group = culture_group:jinyu_group
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.1
+		caster_damage_mult = 0.1
+		caster_toughness_mult = 0.1
+		knight_effectiveness_mult = 0.04
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+
+####################
+##Early Medieval##
+####################
+innovation_saronite_arms = {
```

Comment:
I would like to replace it with an actual building. Just like Africans don't have Gold cultural innovation, but have gold mine buildings in the vanilla.

Reply:
Do we have a building idea sheet or should I make one, I will start working on it soon.

Reply:
> Do we have a building idea sheet where I could carry this over or should I make one, I will start working on that branch soon.

I think you can use [This](https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1906407470&range=A60:A63). Though we don't convert docs cost to in-game gold cost.